samples/mqtt/mqtt5_aws_websocket/README.md

```diff
@@ -78,10 +78,11 @@ cmake --build build --config "Debug"
 
 ## How to run
 
-To Run this sample from the `samples\mqtt\mqtt5_aws_websocket` folder, use the following command:
+To run this sample from the `samples/mqtt/mqtt5_aws_websocket` folder, use the following command:
 
 ```sh
-.mqtt5_aws_websocket \
+cd build
+./mqtt5_aws_websocket \
```

Comment:
Trivial: the description says "To run this sample from the folder". Instead of cd build should we just provide the command `build/mqtt5_aws_websocket` which is to run the sample from the sample folder?

Reply:
I would like to remove the build from the final command as the user might build in a different folder. I updated the instruction to the following. Hope this make more sense. 


> To run this sample, navigate to the build directory where the executable was created:
> 
> ```sh
> # From samples/mqtt/mqtt5_aws_websocket/, go to the build directory
> cd build
> ./mqtt5_aws_websocket \ 
>   --endpoint <AWS IoT endpoint> \
>   --signing_region <Signing region for websocket connection>
> ```
>